### ISSUE TITLE
Make the version of zabbixapi included changable

### ISSUE THREAD

**Issue by bduong** [laradji/zabbix#164]
The version of zabbixapi included in recipes/_providers_common.rb is 0.6.3 which is deprecated according to https://github.com/express42/zabbixapi.

Since users might be using an older version of zabbix, it should be configurable to match the server version.

**Comment by mgolfieri** [comment]
Agreed.  Currently it'd be even better if the chef_gem zabbix api calls supported upgrade and an optional version. If not present, it should upgrade to latest. Currently 2.4.5 is the latest. According to zabbixapi:

"""We support only two last versions of zabbix (2.2 and 2.4), so you should consider zabbixapi 0.6.x and 2.0.x depricated."""

**Comment by chewi** [comment]
See #224.
